### ISSUE TITLE
Now tv

### ISSUE THREAD

**Issue by Seby68** [richardfrost/AdvancedProfanityFilter#464]
Now tv

Hi Richard, a while ago I told you about now tv, this site opens through a browser, but uses a personal player to open the movies. So the application we use doesn't work. I clicked with the right and left mouse button, to see if it behaved like a browser, but I didn't go back to anything, it doesn't open elements or anything else. I'm sending you photos, to see if you can do something? You know I thought so , because if everyone did this , your app. we could no longer use it. Let me know please, bye and thank you.The photos can be found in my Outlook email.

**Comment by richardfrost** [comment]
I found some more details on their site about the player:

- [What devices can I use to watch NOW?](https://help.nowtv.com/article/what-devices-can-i-use-to-watch-now)
- [Watching Now on Windows](https://help.nowtv.com/article/watching-now-on-windows)

Based on that information, and the screenshots and details @Seby68 included in some emails, I don't think there is anything we can do because they are only using the browser to do allow selection of a video to play. After selecting it, streaming is only able to go through that app. I imagine they are doing it to help control access to the videos for security purposes, but it could be several other reasons as well.

In summary, I don't think APF will be able to do anything to allow filtering/muting for Now TV.

**Comment by richardfrost** [comment]
Thanks for the update @robbo90210. Unfortunately I don't have access to Now TV, but if you're willing to help me gather some information it might be possible to add support for it if it is indeed playing in the browser now. If you'd like to help out, please send an email to support @ frostco.dev (without the spaces).
